HelpMyStreetFE/HelpMyStreetFE/Models/Account/Jobs/RequestDetail.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using HelpMyStreet.Utils.Models;
+
+namespace HelpMyStreetFE.Models.Account.Jobs
+{
+    public class RequestDetail
```

Comment:
Same again. Does this need to defined here

Reply:
Good shout.  That class is identical to GetRequestDetailsResponse so I'll remove mine.